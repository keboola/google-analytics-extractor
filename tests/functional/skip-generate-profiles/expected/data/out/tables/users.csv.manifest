{"destination":"out.outputBucket.users","incremental":true,"primary_key":["id"]}