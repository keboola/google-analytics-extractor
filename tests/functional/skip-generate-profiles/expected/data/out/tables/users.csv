"id","idProfile","date","pageviews"
"3916a2752f83c9c8cb99f93bbc186feb5c63347a","184062725","2023-04-01","757"
"c168e938d894693e885a0c24b9d9fd48b023d7aa","184062725","2023-04-02","831"
"fdd493abd73280292218e688222538b1085392cc","184062725","2023-04-03","1727"
"9e2cd912994d5d98521071659443d42fa01c133a","184062725","2023-04-04","1921"
"650c8d82ba8cf7f2cb09e12ad1108f14549011de","184062725","2023-04-05","1754"
"e3b30c0d4d8debe06dd4702afbab388b9b7b4c5a","184062725","2023-04-06","1589"
"ee2b13f04b68b19c39f840542e7598e964d5afb7","184062725","2023-04-07","1010"
"0ef3cbd7521215bf3ca52c11a83d46dcd2e798d9","184062725","2023-04-08","644"
"b0a1f0fc97eab20d48070bc6b9f9897efff2ae23","184062725","2023-04-09","689"
"dc72ec43398d3e6e584b7cb77227414354c1ea35","184062725","2023-04-10","1273"
"950a7d993a7c82e091f7d779c8d93ebfae5c1073","184062725","2023-04-11","1744"
"5ad9eb3466fd4f260ba53163c6cf4130972b89a9","184062725","2023-04-12","1763"
"fda0468c3a520cf054ccb19165f66da2d5d98336","184062725","2023-04-13","1783"
"fc88c429decd300bc4adde77c504d726c62dd4bf","184062725","2023-04-14","1798"
"d0a9697c1ae51d592e9b1f4f43f21470f05bf57f","184062725","2023-04-15","790"
"7519d65b0f9231b2fd60f0bcdbdc39b645029dd7","184062725","2023-04-16","740"
"2bd46e0a3306ecd9f366d3721ea0054458d8f3b1","184062725","2023-04-17","1915"
"5f4ddd6c5910eb0011303f04a06f9ef5b8fb5e88","184062725","2023-04-18","1928"
"0bece15a6990256c7596dc62cda9f480ba6c369b","184062725","2023-04-19","2008"
"f0618e8bb052531b3059e5fe8380cdd1f5063adf","184062725","2023-04-20","2470"
